src/main/java/sysc4806/project24/mini_survey_monkey/controllers/SummaryController.java

```diff
@@ -22,7 +24,15 @@ public SummaryController(SurveyRepository surveyRepository) {
     @GetMapping("/summary/{surveyID}")
     public String viewSurvey(@PathVariable("surveyID") int surveyID, Model model) {
         Survey survey = surveyRepository.findById(surveyID);
+
+        // Retrieve all CommentQuestions for this survey
+        List<CommentQuestion> commentQuestions = survey.getQuestions().stream()
+                .filter(question -> question instanceof CommentQuestion)
+                .map(question -> (CommentQuestion) question)
+                .toList();
+
         model.addAttribute("survey", survey);
+        model.addAttribute("commentQuestions", commentQuestions);
```

Comment:
Is this needed? It doesn't looks like its being used in `summary.html`

Reply:
this can be removed i forgot to remove it before pushing code